JS/Lesson15/sinan/script.js

```diff
@@ -116,14 +129,49 @@ function checkIfStringIsUpperCase(stringToCheck) {
 	}
 }
 
-checkIfStringIsUpperCase('Write a program that takes in');
-checkIfStringIsUpperCase('write a program that takes in');
+checkIfStringIsUpperCase('Exercise 4.1:','Write a program that takes in');
+checkIfStringIsUpperCase('Exercise 4.2:','write a program that takes in');
 
 // 5. Write a program that takes in a number and checks if it's positive, negative, or zero using if else statement.
+function checkNumberPositive(checkNumber) {
+	if (checkNumber > 0){
+		return 'checkNumber is positive';
+	}
+	else if (checkNumber < 0){
+		return 'checkNumber is negative';
+	}
+	else if (checkNumber === 0){ 
+		return 'checkNumber is equal to zero';
+	}
+	else {
+		return 'NaN';
+	}
+}
+console.log('Exercise 5:',checkNumberPositive(-55));
+
 
 // 6. Write a program that takes in a year and checks if it's a leap year (is divisible by 4 and not divisible by 100 OR divisible by 400) or not using if else statement.
+function LeapYear(checkYear) {
+	if (checkYear %4 === 0){
+		return 'Leap Year';
+	}
+	else if (!checkYear %4 === 0){
+		return 'Not Leap Year';
+	}
+	
+}
+console.log('Exercise 6:',LeapYear(2024));
 
 // 7. Write a program that takes in a temperature and checks if it's above or below freezing using if else statement.
+function checkTemperature(freezingPoint) {
+	if (freezingPoint <= 0){
```

Comment:
0 is actually freezing point so it should not be included
```suggestion
	if (freezingPoint < 0){
```